build-scripts/generate_man_page.py

```diff
@@ -10,7 +10,10 @@
 import os
 import io
 
-import xml.etree.ElementTree as ET
+try:
+    from xml.etree import cElementTree as ElementTree
+except ImportError:
+    import cElementTree as ElementTree
```

Comment:
?!?

Just import ssg.xml and get the ElementTree from there. Definitely don't copy paste code!

https://github.com/ComplianceAsCode/content/blob/master/ssg/xml.py

Reply:
Aha, I'll do that. I'll provide a comment to prevent readability problems.